### DISCUSSION TITLE
Set Conditional Formatting DataBar Border Color

### DISCUSSION THREAD

**Discussion by geoced** [dfinke/ImportExcel#1488] [Q&A]
Hello,

I'm using `Add-ConditionalFormatting -Address $range -DataBarColor 'Red' -PassThru` to set a data bar color on some cells, but compared to the manual way of doing using Excel's GUI, the difference is that the data bar color is left unchanged by ImportExcel, while Excel sets it to a slightly darker color than the data bar gradient color. 

Excel GUI
![image](https://github.com/dfinke/ImportExcel/assets/40609578/eeaaafca-b548-489e-8e90-51cf4a0835c6)

ImportExcel
![image](https://github.com/dfinke/ImportExcel/assets/40609578/1de559c8-707c-460c-ab05-2840715c2a41)

Is there a way to set the data bar border color or is this not supported by EPPlus ?

Thanks in advance !

**Comment by dfinke** [comment]
Thanks. That is an EPPlus question. The version I'm using may not support this. Worth searching.

**Reply by geoced** [reply]
Hello Doug,
Just to let you know that version 7.0 of EPPlus supports a whole range of additionnal settings for databars, including databar border color,  as described [here](https://github.com/EPPlusSoftware/EPPlus/wiki/Conditional-Formatting-%E2%80%90-Databars).
Which version of EPPlus is bundled with the latest version of the ImportExcel PowerShell module ?
Thanks !

**Reply by dfinke** [reply]
Yes. The PS module uses version 4.x.  The new versions had many breaking changes and in most cases requires a paid license.
